apps/condo/domains/common/utils/helpers.ts

```diff
@@ -1,7 +1,23 @@
+import { ParsedUrlQuery } from 'querystring'
+import get from 'lodash/get'
+
 export const formatPhone = (phone?: string): string => {
     if (!phone) {
         return phone
     }
 
     return phone.replace(/(\d)(\d{3})(\d{3})(\d{2})(\d{2})/, '$1 ($2) $3-$4-$5')
```

Comment:
Provide example in comments, it's hard to understand what it does.

Reply:
What to extract? Regular expression? I'm afraid this will break the connection with the second argument and impair understanding. Made a comment with an example.